.travis.yml

```diff
@@ -41,3 +41,5 @@ jobs:
         - BEHAT_TEST=1
         - REQUIRE_GRAPHQL="^4@dev"
         - BEHAT_SUITE="cms"
+    - env:
+        - NPM_TEST=1
```

Comment:
now that the provision has been updated we should be able to remove these lines

Reply:
:facepalm: